src/integrations.py

```diff
@@ -127,12 +135,18 @@ def base_dn(self) -> str:
     def starttls_enabled(self) -> bool:
         return self._charm.config.get("starttls_enabled", True)
 
+    @property
+    def ldap_over_tls_enabled(self) -> bool:
+        return self._charm.config.get("ldap_over_tls_enabled", True)
```

Comment:
The default value in `charmcraft.yaml` is `false`, but here we set it to `True`. I think that the default value here will never be used (?), but let's keep it consistent.

Reply:
Fixed in 0a2e11b